pyproject.toml

```
@@ -36,6 +36,7 @@ dependencies = [
   "cryptography>=42.0.2",
   "cssselect>=1.2,<1.3",
   "Cython>=3.0.0,<3.1",
+  "cyrtranslit>=1.1.0",
```

Please add upper bound as well to make sure new versions are CI tested before accepted.

```suggestion
  "cyrtranslit>=1.1.0,<2.0",
```